package.json

```
@@ -27,6 +27,7 @@
     "@babel/preset-env": "^7.16.11",
     "babel-loader": "^8.2.4",
     "copy-webpack-plugin": "^10.2.4",
+    "dayjs": "^1.11.2",
     "webpack": "^5.72.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.8.1"
```

Я уже говорил, повтрюсь: мы должны фиксировать версии без модификаторов
```suggestion
    "@babel/preset-env": "7.16.11",
    "babel-loader": "8.2.4",
    "copy-webpack-plugin": "10.2.4",
    "dayjs": "1.11.2",
    "webpack": "5.72.0",
    "webpack-cli": "4.9.2",
    "webpack-dev-server": "4.8.1"
```